apps/nameguard.io/src/app/layout.tsx

```diff
@@ -18,9 +18,13 @@ const title = "Protect your community with NameGuard for ENS";
 const description =
   "Guard your users from heartbreak and encourage best practice usage of ENS.";
 
+const productionUrl =
```

Comment:
This doesn't work the way you've assumed it would. If this environment variable isn't defined it will result in the following value for `productionUrl`: "https://undefined".

Please make sure to test logic like this. Thanks